packages/next/build/swc/src/styled_jsx/transform_css.rs

```
@@ -187,6 +187,10 @@ impl Namespacer {
                                 .cloned()
                                 .collect::<Vec<_>>();
 
+                            if node.combinator.is_some() {
+                                v[0].combinator = node.combinator;
```

Can be `v[0].combinator` some?
If so, it can silently swallow an error.

---

Thanks! Then it's fine.